ui/app/mainui/StatusTrayIcon.qml

```diff
@@ -6,13 +6,16 @@ SystemTrayIcon {
     id: root
 
     property bool isProduction: true
+    property bool showRedDot: false
 
     signal activateApp()
 
     visible: true
+
+
     icon.source: Qt.platform.os === Constants.windows // TODO: Add status-logo-white with stroke for windows
```

Comment:
Hmm so this is not gonna work for Windows... what was the problem there @alexjba ?

Reply:
We didn't had proper assets for windows
https://github.com/status-im/status-desktop/issues/14855

Reply:
Alisher provided the proper tray icon with and without the red dot in this task https://github.com/status-im/status-desktop/issues/14855

Reply:
Oh nice. I can update the Windows icon in this PR then. Do you know if I can find the SVG somewhere?

Reply:
Ah actually, it uses png for Windows. I can just use those then!